src/Elmah.Io.NLog/ElmahIoTarget.cs

```diff
@@ -87,8 +83,16 @@ private string User(LogEventInfo logEvent)
         }
 
         private List<Item> PropertiesToData(IDictionary<object, object> properties)
-        {
-            return properties.Keys.Select(key => new Item{Key = key.ToString(), Value = properties[key].ToString()}).ToList();
+        {

```

Comment:
It's funny that you need to change this. I saw it in your code as well, but I didn't have any problems with the code above in my ASP.NET Core sample.